_docs/datadestinations/google-analytics-costs.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: page
+section: datadestinations
+title: "Google Analytics Costs"
+order: 1
+---
+
+In this section you will learn how to configure export of advertisement costs to Google Analytics.
+
+Google Analytics Data Destination exports Data Sources costs data to Google Analytics.
+After enabling export you would be able to see cost data in Google Analytics Acquisition report.
+
+> Attention! The [Google BigQuery](/integrations/google-bigquery) integration has to be enabled to use this feature.
+
+### Page contents
+------
+<ul class="page-navigation">
+  <li><a href="#importing-data">Importing data from costs table in BigQuery</a></li>
+  <li><a href="#connecting-and-configuring">Connecting and configuring</a></li>
+  <li><a href="#create-dataset">Create Cost Data Data Set in Google Analytics</a></li>
+  <li><a href="#data-destination-setup">Setting up Data Destination in SegmentStream</a></li>
+  <li><a href="#options-description">Options description</a></li>
+  <li><a href="#display-name">Display name</a></li>
+  <li><a href="#account-id">Account ID</a></li>
+  <li><a href="#tracking-id">Tracking ID</a></li>
+  <li><a href="#data-set-type">Data Set Type</a></li>
+  <li><a href="#data-set-id">Data Set ID</a></li>
+  <li><a href="#saving-disconnecting-disabling">Saving, Disconnecting, Disabling data source</a></li>
+  <li><a href="#where-to-get-leads-data">Where to get leads data</a></li>
+  <li><a href="#table-structures">Table structures</a></li>
+</ul>
+
+### <a name="importing-data"></a>Importing data from costs table in BigQuery
+------
+
+After enabling this data destination, costs data will be uploaded from allCosts (costsFinal) table in BigQuery to Google Analytics once every 24 hours.
+
+### <a name="connecting-and-configuring"></a>Connecting and configuring
+------
+
+#### <a name="create-dataset"></a>Create Cost Data Data Set in Google Analytics
+1. Sign in to Google Analytics.
+2. Click Admin, and navigate to the property to which you want to upload the data.
+3. In the PROPERTY column, click Data Import. This displays the Data Sets page.
+4. Choose "Create"
+5. Select "Cost data" Data Set type
+6. Configure Data Set schema:
+
+    a. In **At least one of this columns must be provided:** choose following fields:
+    - Clicks
+    - Costs
+    - Impressions
+
+
+    b. In **Any number of these columns may be provided:** choose following fields:
+    - Ad content
+    - Campaign
+    - Keyword
+
+    c. Set **Import behaviour** field to **Overwrite**
+
+
+Read more in [Google Analytics documentation on Data Import](https://support.google.com/analytics/answer/3191589?hl=en#summary_data_import)
+
+#### <a name="data-destination-setup"></a>Setting up Data Destination in SegmentStream
+The process of connecting data destinations is similar to [connecting data sources](/datasources/index).
```

Comment:
I think reference here to data sources it's not good.
Because as user of docs I like read one page instead  jumping between pages.